### DISCUSSION TITLE
Should we remove "Include earning variation" from the household calculator Reproduce in Python page?

### DISCUSSION THREAD

**Discussion by anth-volk** [PolicyEngine/policyengine-app#1406] [Q&A]
I'm kind of curious what its original purpose was and if it's a holdover from a previous version

**Comment by MaxGhenis** [comment]
It adds axes such that results show many different values corresponding to earnings levels. It helps reproduce the "Varying your earnings" chart but distracts otherwise, when the user just wants one result. So I think it's worthwhile.

**Reply by anth-volk** [reply]
I wonder if there's any way - and any benefit, for that matter - to expanding out the functionality to cover all variables included in the household, then? It could be as simple as iterating over the household to determine what variables exist (excluding basic inputs), and then adding a dropdown? Just a thought, not sure if the user base would benefit from such a feature

**Reply by MaxGhenis** [reply]
Oh like "Vary by" and a drop down? Yeah that'd be great for the Varying your earnings chart (eg vary dividends) and we could have it sync to the Reproduce in python page.

**Reply by anth-volk** [reply]
Okay. I'll have to think through this and mock something up either this week or next, depending on a few other, higher-priority items
